mod-audit-server/src/main/java/org/folio/services/acquisition/impl/PieceAuditEventsServiceImpl.java

```diff
@@ -0,0 +1,48 @@
+package org.folio.services.acquisition.impl;
+
+import io.vertx.core.Future;
+import io.vertx.pgclient.PgException;
+import io.vertx.sqlclient.Row;
+import io.vertx.sqlclient.RowSet;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.dao.acquisition.PieceEventsDao;
+import org.folio.kafka.exception.DuplicateEventException;
+import org.folio.rest.jaxrs.model.PieceAuditEvent;
+import org.folio.rest.jaxrs.model.PieceAuditEventCollection;
+import org.folio.services.acquisition.PieceAuditEventsService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class PieceAuditEventsServiceImpl implements PieceAuditEventsService {
+  private static final Logger LOGGER = LogManager.getLogger();
+  private static final String UNIQUE_CONSTRAINT_VIOLATION_CODE = "23505";
```

Comment:
lets move to AuditEventDBConstants for all 3 our services

Reply:
Changed